build.ps1

```diff
@@ -46,6 +46,8 @@ function bazel() {
 # which is a workaround for this problem.
 bazel shutdown
 
+bazel clean --expunge
```

Comment:
Maybe add a comment that this is a workaround and link to https://github.com/digital-asset/daml/issues/4162, so we don't forget to fix it.

Reply:
I’ll add it separately, I want to merge this sooner rather than later :slightly_smiling_face:

Reply:
:+1: